src/gherkin/index.js

```diff
@@ -205,8 +201,8 @@ class Step {
 }
 
 function verifyMatchParams(pattern) {
-  var reg = RegExp(/(\d+|"([^\"]*)"|'([^\']*)')/g) // eslint-disable-line no-useless-escape
-  var data = []
+  const reg = RegExp(/(\d+|"([^\"]*)"|'([^\']*)')/g) // eslint-disable-line no-useless-escape
+  const data = []
```

Comment:
Note: it feels strange, but seems to be the way of modern JS. An array can be declared `const` and `.push` can be used to add elements to the array. The constant thing is the pointer to the array, which is what is actually stored in the variable.

My IDE really wants me to code that way, even though it feels strange to me.